apps/website-new/docs/en/guide/start/quick-start.mdx

```diff
@@ -98,11 +98,11 @@ import { pluginReact } from '@rsbuild/plugin-react';
 import { pluginModuleFederation } from '@module-federation/rsbuild-plugin';
 
 export default defineConfig({
-  plugins: [pluginReact()],
```

Comment:
The "plugins" key in the configuration object was duplicated and the code didn't work. Unifying the two in one makes it work.